src/app/migrations/password/password.1.core.txt

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <form clrForm>
+      <cds-password labelLayout="hidden-label" control-width="shrink">
+        <label>input</label>
+        <input type="password" placeholder="input" name="input" [(ngModel)]="input" />
```

Comment:
`placeholder="Type a password"` - same for all `placeholders`

Reply:
I don't see what is bothering you here but if you want to change it I will.